tests/updateAMM.ts

```diff
@@ -0,0 +1,405 @@
+import * as anchor from '@project-serum/anchor';
+import { assert } from 'chai';
+import { BN, getMarketOrderParams, OracleSource, ZERO } from '../sdk';
+
+import { Program } from '@project-serum/anchor';
+
+// import { PublicKey } from '@solana/web3.js';
+
+import {
+	Admin,
+	MARK_PRICE_PRECISION,
+	calculateMarkPrice,
+	calculateTradeSlippage,
+	PositionDirection,
+	EventSubscriber,
+	convertToNumber,
+	findComputeUnitConsumption,
+	calculateBidAskPrice,
+	calculatePrepegAMM,
+} from '../sdk/src';
+
+import {
+	getFeedData,
+	// initUserAccounts,
+	mockOracle,
+	mockUserUSDCAccount,
+	mockUSDCMint,
+	setFeedPrice,
+	getOraclePriceData,
+	initializeQuoteAssetBank,
+} from './testHelpers';
+
+describe('update amm', () => {
```

Comment:
What are all the test cases we want?
1) amm balanced, oracle above peg
2) amm balanced, oracle below peg
3) amm unbalance, oracle below peg, sufficient fees
4) amm unbalance, oracle above peg, sufficient fees
5) amm unbalance, oracle below peg, insufficient fees
6) amm unbalance, oracle below peg, insufficient fees
7+) update multiple markets at once